support-files/mariadb.service

```diff
@@ -0,0 +1,104 @@
+#
+#  /etc/systemd/system/mariadb.service
+#
+
+[Unit]
+Description=MariaDB database server
+After=network.target
+After=syslog.target
```

Comment:
That's different. "dbms shouldn't use socket activation because they are slow to start" applies to dbms using socket activation. Here I mean that syslog uses socket activation, so anything that uses syslog can start right away without explicit "After", because socket activation (of syslog, not dbms!) will automatically guarantee the correct ordering.

Not sure about After=network.target though.